.travis.yml

```diff
@@ -20,3 +19,15 @@ notifications:
 after_success:
   - julia -e 'using Pkg; Pkg.add("Coverage"); Pkg.dir("FFTW"); using Coverage; Coveralls.submit(Coveralls.process_folder())'
   - julia -e 'using Pkg; ps=Pkg.PackageSpec(name="Documenter", version="0.19"); Pkg.add(ps); Pkg.pin(ps); Pkg.dir("FFTW"); include(joinpath("docs", "make.jl"))'
+
+jobs:
+  include:
+    - stage: "Documentation"
+      julia: 1.0
+      os: linux
+      script:
+        - julia --project=docs/ -e 'using Pkg; Pkg.instantiate();
+                                               Pkg.develop(PackageSpec(path=pwd()));
```

Comment:
`develop` should come before `instantiate`, otherwise weird version resolution issues can happen

Reply:
Guys, I can't follow all [changes to documentation](https://juliadocs.github.io/Documenter.jl/v0.20/man/hosting/#.travis.yml-Configuration-1) :sweat_smile:

Reply:
That should probably be updated; I'm not sure why it's like that.

Reply:
It's updated in the latest version (I linked docs for v0.20), but I copied from another Travis script of mines that was done some time ago.